tests/RobotFramework/tests/cumulocity/registration/device_deregistration.robot

```diff
@@ -0,0 +1,103 @@
+*** Settings ***
+Resource    ../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO
+
+Test Tags    theme:c8y    theme:registration
+Suite Setup    Custom Setup
+Test Setup    Test Setup
+Test Teardown    Get Logs    ${DEVICE_SN}
+
+*** Test Cases ***
+
+Main device registration
+    ${mo}=    Device Should Exist              ${DEVICE_SN}
+    ${mo}=    Cumulocity.Device Should Have Fragment Values    name\=${DEVICE_SN}
+    Should Be Equal    ${mo["owner"]}    device_${DEVICE_SN}
+    Should Be Equal    ${mo["name"]}    ${DEVICE_SN}
+
+
+Child device registration
```

Comment:
```suggestion
Child device deregistration
```
Instead of duplicating these tests from the `device_registration.robot`, how about adding the de-registration steps to the existing tests themselves?

Reply:
I did this at the beginning and would preffer that approach, my intention was: as soon we have good solution for automating the deregistration to merge the test steps to one test.